test/unit/executionPlanWebviewController.test.ts

```diff
@@ -229,11 +249,15 @@ suite("ExecutionPlanWebviewController", () => {
 
         assert.deepStrictEqual(
             updateTotalCostStub.firstCall.args,
-            [mockState, mockPayload],
+            [mockInitialState, mockPayload],
             "showQuery should be called with correct arguments",
         );
 
-        assert.deepStrictEqual(result, mockResultState);
+        assert.deepStrictEqual(
+            result,
+            mockResultState,
+            "State should be updated by reducer call",

```

Comment:
Even better would be to say _how_ the state should've been updated (e.g. "state should have totalCost updated")